tests/testthat/test-lancelot-support.R

```diff
@@ -554,6 +565,14 @@ test_that("lancelot_compare combines likelihood correctly", {
     hosp = 36,
     deaths_carehomes = 4,
     deaths_hosp = 5,
+    deaths_hosp_0_49 = 5,
+    deaths_hosp_50_54 = 5,
+    deaths_hosp_55_59 = 5,
+    deaths_hosp_60_64 = 5,
+    deaths_hosp_65_69 = 5,
+    deaths_hosp_70_74 = 5,
+    deaths_hosp_75_79 = 5,
+    deaths_hosp_80_plus = 5,
```

Comment:
again, might want to have different values for each of the above

Reply:
Done